ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -177,10 +176,9 @@
 			};
 			buildConfigurationList = 97C146E91CF9000F007C117D /* Build configuration list for PBXProject "Runner" */;
 			compatibilityVersion = "Xcode 9.3";
-			developmentRegion = en;
+			developmentRegion = "pt-BR";
```

Comment:
Changed development region because it's protected, and the development region cannot be removed. So I replaced to the pt-BR to remove the english localization